ARS/src/main/java/com/CSC450/ars/controller/RatiosCal.java

```diff
@@ -0,0 +1,24 @@
+package com.CSC450.ars.controller;
+
+public class RatiosCal 
```

Comment:
This class is redundant.  We can completely get rid of this file since the ratio formula calculates the value of the AdLocationVisit object and is defined on that class.